composer.json

```diff
@@ -13,7 +13,7 @@
         }
     },
     "require": {
-        "php": ">=5.5",
+        "php": ">=5.6",
```

Comment:
Ah, reading http://php.net/manual/en/wrappers.php.php#wrappers.php.input , I thought it was not available in <= PHP 5.5. I see I was mistaken. Thanks! I'll update.